modules/apigee_edge_teams/src/Controller/TeamMembersList.php

```diff
@@ -160,11 +160,14 @@ protected function buildRow(string $member, array $users_by_mail, array $team_me
       $row['data']['member'] = $member;
     }
 
+    // Default role assigned to all users.
+    $default_roles = [TeamRoleInterface::TEAM_MEMBER_ROLE => $this->t('Member')];
```

Comment:
I like this approach. However we do need to fetch the `TeamRoleInterface::TEAM_MEMBER_ROLE` label instead of hard-coding it to "member", because the role name might have been edited by an administrator to be something else.